pages/client/index.tsx

```diff
@@ -44,13 +40,9 @@ function Client() {
     Promise
       .all([
```

Comment:
Can we remove the Promise.all since there is now just one item in the array?

Reply:
Won't fix, this PR is only address removing.  A subsequent PR that will address this.